server/db/models/message.js

```diff
@@ -10,6 +10,17 @@ const Message = db.define("message", {
     type: Sequelize.INTEGER,
     allowNull: false,
   },
+  readStatus: {
+    type: Sequelize.BOOLEAN,
+    defaultValue: false,
+  },
+  readById: {
+    type: Sequelize.STRING,
+    defaultValue: null,
+  },
+  readTime: {
+    type: Sequelize.DATE,
+  },
```

Comment:
- Is it necessary to store the id of who each message has been read by? Since this is a 1-1 chat, the only possibility is the other user of the conversation so this field is redundant. 
- Do we really need the date/time that each message was read?

Reply:
noted !